### DISCUSSION TITLE
Not able to get expected response by Rest API.

### DISCUSSION THREAD

**Discussion by Anuragbisenn** [apache/airflow#25497] [Q&A]
i am using [https://airflow.apache.org/api/v1/dags/{dag_id}/dagRuns/{dag_run_id}/taskInstances/{task_id}/{map_index}](https://airflow.apache.org/api/v1/dags/%7Bdag_id%7D/dagRuns/%7Bdag_run_id%7D/taskInstances/%7Btask_id%7D/%7Bmap_index%7D) but i got the error related to {map_index} parameter. it is an integer value like -1, but does not get the response which is expected.
same things happening with other Rest API's which is taking integer as an path parameter.

**Comment by potiuk** [comment]
I tihnk you need to explain what "does not get the response which is expected.". Is it an error? wrong data? empty data? none of the above? All of the above? 

some example here would be handy and more explanation what data you have in your database, what exactly query you run what response you get and what response you expect to get. 

Otherwise, you rob yourself from the opportunity that someone will be able to help you.
